src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -234,6 +235,14 @@ def raid_items(md)
         CWM::PagerTreeItem.new(page)
       end
 
+      # @return [CWM::PagerTreeItem]
+      def bcaches_items
+        devices = device_graph.bcaches
+        page = Pages::Bcaches.new(devices, self)
+        children = devices.map { |v| disk_items(v) }
+        CWM::PagerTreeItem.new(page, children: children, icon: Icons::LVM) # TODO: new nice icon
```

Comment:
Same than above, this should be `Icons::BCACHE`. No matter if we have not found a proper icon, we should define the constant already.